feature/interface/holdtime/otg_tests/holdtime_test/README.md

```diff
@@ -0,0 +1,70 @@
+# RT-5.5: Interface hold-time
+
+## Summary
+
+Verifi configurability of interface hold-time down of 300ms  and hold-time up of 5 sec.\
+Verify oper-state behaviours
+
+## Procedure
+*   Configure DUT port-1 to OTG port-1
+*   Configure static LAG on DUT and OTG with port-1 as member
+*   Configure hold-time down 300ms and hold-time up 5000ms
+### TC1 - configuration verification:
+*   Get hold-time state from device and check if it matches what was send in configuration. (some implementation may round-up/round-down values)
+### TC2 - long down:
+*   Read timestamp of last oper-status change  form DUT port-1 
+*   Start sending Ethernet Remote Fault (RF) from OTG port-1 (or other mean which disable laser on OTG); read and store timestamp form OTG of this operation (OTG_STATE_CHANGE_TS).
+*   wait 1000 ms
+*   Read timestamp of last oper-status change  form DUT port-1 (DUT_LAST_CHANGE_TS)
+*   Verify that DUT LAG:
+  * oper-status is DOWN
+  * oper-status last change time has changed 
+  * DUT_LAST_CHANGE_TS = OTG_STATE_CHANGE_TS + 300ms +/- tolerance; Use tolerance of 200ms.
```

Comment:
maybe for this event our tolerance should be smaller? 100? (if ixia can be that precise at all)